scripts/example.ts

```diff
@@ -0,0 +1,43 @@
+import { join } from "path";
+
+import firebase = require("../src");
+
+const client: any = firebase;
+
+process.on("unhandledRejection", (err) => {
+  console.error("unhandledRejection", err);
+  process.exit(1);
+});
+
+async function listProjects() {
```

Comment:
It's fine now because it's simple, but any reason not to write this file as an actual `mocha` test suite with assertions and stuff?

Reply:
I thought about this, and I was going to be curious what you said. I'll take a look and revamp it with Mocha before submitting.

Reply:
This is written with Mocha now!